### PR TITLE
macro even/odd style on line or column

### PR DESCRIPTION
**Pull request by dvdn**
Macro calendar evo to get right css classes for coloring lines or columns

### CODE REVIEW

**Comment by dvdn** [comment]
Related to https://github.com/CanalTP/TimeTableModels/pull/1
Hi @vinzentt, please tell us if you see possible regression for this.

File: Resources/views/Layouts/macros.html.twig [hunk 1]
```diff
@@ -160,9 +160,9 @@
         </tr>
     {% endif %}
     {% for i in 0..loopMax - 1 %}
-        <tr class="{% if i % 2  == 0 %}even{% else %}odd{% endif %}">
+        <tr {% if colorAltern == 'line' %}class="{% if i % 2  == 0 %}even{% else %}odd{% endif %}"{% endif %}>
         {% for index,hour in hours_range %}
-            <td class="timegrid-column">{% spaceless %}
+            <td class="timegrid-column{% if colorAltern == 'column' %}{% if index % 2  == 0 %} even{% else %} odd{% endif %}{% endif %}">{% spaceless %}
```
Comment (datanel) [inline Resources/views/Layouts/macros.html.twig]: odd and even check is repeated above. you can do it once and put the result in a variable

Comment (datanel) [inline Resources/views/Layouts/macros.html.twig]: you can even use cycle twig function I think

**Comment by datanel** [comment]
How do I test this PR?

**Comment by dvdn** [comment]
'Blocked' by CanalTP/TimeTableModels#1

File: Resources/views/Layouts/macros.html.twig (hunk 1 above)
Comment (datanel) [inline Resources/views/Layouts/macros.html.twig]: {{ cycle(['odd', 'even'], index) }}

**Comment by dvdn** [comment]
Nice contrib :+1:

**Comment by vinzentt** [comment]
No i don't see any regression, maybe with frequencies but I don't remember how frequencies are rendered regarding this.

File: Resources/views/Layouts/macros.html.twig [hunk 2]
```diff
@@ -160,9 +160,9 @@
         </tr>
     {% endif %}
     {% for i in 0..loopMax - 1 %}
-        <tr class="{% if i % 2  == 0 %}even{% else %}odd{% endif %}">
+        <tr {% if colorAltern == 'line' %}class="{{ cycle(['even', 'odd'], i) }}"{% endif %}>
```
Comment (vinzentt) [inline Resources/views/Layouts/macros.html.twig]: why not simply:
{% set cssClass = i % 2 == 0 ? 'even' : 'odd' %}

Comment (Netmisa) [inline Resources/views/Layouts/macros.html.twig]: +1

**Pull request merged**